virt/virtual_machines/virt-create-vms.adoc

```diff
@@ -53,6 +53,10 @@ To configure storage class defaults, use storage profiles. For more information,
 
 include::modules/virt-creating-vm-yaml-web.adoc[leveloffset=+2]
 
+include::modules/virt-creating-vm-instance-type.adoc[leveloffset=+1]
+
+include::modules/virt-golden-images.adoc[leveloffset=+2]
```

Comment:
Golden images are actually part of auto-updated boot sources documentation in the VM templates folder. 

We use templates to create VMs but we have a separate folder for VM templates.